src/scenes/PlantsDashboardRouter/components/MultiplePlantingSiteMap.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { CSSProperties, useEffect, useMemo, useState } from 'react';
+
+import { Box, CircularProgress } from '@mui/material';
+
+import { PlantingSiteMap } from 'src/components/Map';
+import { useLocalization } from 'src/providers';
+import { MapService, TrackingService } from 'src/services';
+import { PlantingSite } from 'src/types/Tracking';
+
+type MultiplePlantingSiteMapProps = {
+  projectId: number;
+  organizationId: number;
+  hideAllControls?: boolean;
+  style?: CSSProperties;
+};
+
+export default function MultiplePlantingSiteMap({
+  projectId,
+  organizationId,
+  hideAllControls,
+  style,
+}: MultiplePlantingSiteMapProps): JSX.Element {
+  const { activeLocale } = useLocalization();
+  const [plantingSites, setPlantingSites] = useState<PlantingSite[]>();
+
+  useEffect(() => {
+    const populatePlantingSites = async () => {
+      const serverResponse = await TrackingService.listPlantingSites(organizationId, undefined, activeLocale);
```

Comment:
Planting sites should be get from new planting site provider, will modify when that is ready